src/coreclr/vm/jithelpers.cpp

```diff
@@ -5496,8 +5497,9 @@ void F_CALL_CONV HCCALL1(JIT_ReversePInvokeEnter, ReversePInvokeFrame* frame)
     INSTALL_EXCEPTION_HANDLING_RECORD(&frame->record.m_ExReg);
 #endif
 }
+HCIMPLEND
 
-void F_CALL_CONV HCCALL1(JIT_ReversePInvokeExitTrackTransitions, ReversePInvokeFrame* frame)
+HCIMPL1(void, JIT_ReversePInvokeExitTrackTransitions, ReversePInvokeFrame* frame)
```

Comment:
For consistency, same for the two exit helpers as well (since they don't fulfill the contract on exit).